src/richie-front/js/data/configureStore.ts

```diff
@@ -1,16 +1,25 @@
-import { applyMiddleware, createStore } from 'redux';
+import { applyMiddleware, compose, createStore } from 'redux';
 import createSagaMiddleware from 'redux-saga';
 
+import { Config } from '../types/config';
 import { rootReducer, RootState } from './rootReducer';
 import { rootSaga } from './rootSaga';
 
 const sagaMiddleware = createSagaMiddleware();
 
+declare const CONFIG: Config;
+
 export function configureStore(preloadedState: RootState) {
+  const composeEnhancers =
+    CONFIG.enableDevTools &&
+    (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+      ? (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
```

Comment:
I think `&&` and `||` would be clearer here. This relies on operator precedence which might be confusing for some